src/EventStore.Core/TransactionLog/Scavenging/Interfaces/IChunkReaderForAccumulator.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System.Collections.Generic;
+using EventStore.Core.Helpers;
+
+namespace EventStore.Core.TransactionLog.Scavenging {
+	public interface IChunkReaderForAccumulator<TStreamId> {
+		IEnumerable<RecordForAccumulator<TStreamId>> ReadChunk(
```

Comment:
suggestion: could be renamed to ReadChunkInto() which is a bit more intuitive (a bit how like it's done for the IChunkReaderForExecutor's ReadInto() method)

Reply:
done